multus/multus.go

```diff
@@ -107,18 +57,21 @@ func consumeScratchNetConf(containerID, dataDir string) ([]byte, error) {
 	return ioutil.ReadFile(path)
 }
 
-func getifname() (f func() string) {
-	var interfaceIndex int
-	f = func() string {
-		ifname := fmt.Sprintf("net%d", interfaceIndex)
-		interfaceIndex++
-		return ifname
+func getIfname(delegate *types.DelegateNetConf, argif string, idx int) string {
+	if delegate.IfnameRequest != "" {
+		return delegate.IfnameRequest
+	}
+	if delegate.MasterPlugin {
```

Comment:
I am getting in some case of using the conf file name alone in the CRD, the mode 3. Error `network: cannot set "flannel" ifname to "eth0": ifname eth0 is already exist`